apps/src/templates/sectionsRefresh/QuickAssignTable.jsx

```diff
@@ -15,10 +15,42 @@ export default function QuickAssignTable({
   courseOfferings,
   setSelectedCourseOffering,
   updateCourse,
-  sectionCourse
+  sectionCourse,
+  isNewSection
 }) {
   // Key is type of curriculum e.g. 'Course' or 'Module', which is the singular
   // version of the title we want for the column
+
+  useEffect(() => {
```

Comment:
this seems like an ok place to put the `useEffect` to me? i am curious what broke when you put it in the parent component, but I don't see a strong reason that makes me think it should be in one place over the other